test-clients/src/main/java/com/hedera/services/bdd/suites/consensus/TopicUpdateSuite.java

```diff
@@ -82,6 +82,11 @@ public static void main(String... args) {
 		);
 	}
 
+	@Override
+	public boolean canRunAsync() {
+		return true;
+	}
+
```

Comment:
Should we also change `new TopicUpdateSuite().runSuiteSync();` to `new TopicUpdateSuite().runSuiteAsync();`?

Reply:
when running through suiteRunner we directly call the method `HapiApiSuite::runSuiteAsync` on suites which are configured to run async and vice-verse. We dont use `main` method.